data-pipeline/derived-de-duplication/src/main/java/org/ekstep/ep/samza/service/DeDuplicationService.java

```diff
@@ -30,7 +30,6 @@ public void process(DeDuplicationSource source, DeDuplicationSink sink) throws E
 			if (checksum == null) {
 				LOGGER.info(event.id(), "EVENT WITHOUT CHECKSUM & MID, PASSING THROUGH : {}", event);
 				event.markSkipped();
-				event.updateDefaults(config);
```

Comment:
Any reason for this to be removed?

Reply:
We don't need to update the default channel in all the downstream jobs and hence this code was removed.